packages/integration-tests/src/test-projects/data-storage-auth-with-triggers-ts/amplify/func-src/response_generator.ts

```diff
@@ -1,13 +1,14 @@
 import { Amplify } from 'aws-amplify';
 import { downloadData, uploadData } from 'aws-amplify/storage';
+import { env } from '@env/defaultNodeFunction.js';
```

Comment:
Chose `defaultNodeFunction` and added typedef file for this function since `node16Function` also defines the same environment variables. We would have to rethink this or have a `response_generator.ts` for each function if we want this to test typedef files for multiple functions or if these two functions start to differ for environment variables at runtime.